ocaml/xapi/export.ml

```diff
@@ -467,6 +466,33 @@ let make_pvs_sites table __context self =
   ; snapshot= API.rpc_of_pVS_site_t site
   }
 
+let make_vtpm table __context self =
+  debug "exporting vtpm %s" (Ref.string_of self) ;
+  let lookup' ref = lookup table (Ref.string_of ref) in
+  let vtpm = Db.VTPM.get_record ~__context ~self in
+  let secret =
```

Comment:
Can we use the API call implementation here? Then when we implement etcd it'll all be transparent to this code.
There is some special handling there for unique vtpms, which we'll need to think about for export. (I think the REQ has some details how to handle those)